memory.x

```diff
@@ -6,6 +8,13 @@ MEMORY
   RAM3 (xrw)      : ORIGIN = 0x20020000, LENGTH = 64K
   CCMRAM (rw)     : ORIGIN = 0x10000000, LENGTH = 64K
 }
-
 _stack_start = ORIGIN(RAM) + LENGTH(RAM);
 _heap_size = LENGTH(RAM) - 4K;
+*/
+
+/* STM32F107 */
+MEMORY
+{
+  FLASH : ORIGIN = 0x08000000, LENGTH = 128K
+  RAM : ORIGIN = 0x20000000, LENGTH = 32K
+}
```

Comment:
I think is should be reverted default `memory.x` to define STM32F429 and commented for STM32F107. Or there should be another file `memory_stm32f107.x`